reporter/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -194,8 +194,11 @@ class CopyrightStatementsProcessor {
     private fun replaceYears(copyrightStatement: String, placeholder: String): Pair<String, Set<Int>> {
         val resultYears = mutableSetOf<Int>()
 
-        val replaceRangeResult = replaceAllYearRanges(copyrightStatement, placeholder)
-        var currentStatement = replaceRangeResult.first
+        // Fix up strings containing e.g.: 'copyright u'2013'
+        var currentStatement = copyrightStatement.replace("(?=.*\\b)(u')(?=\\d{4}\\b)".toRegex(), "")
```

Comment:
Personally, I find these "zero-width positive lookahead" regexes very hard to read. Why not use a replacement with back-references instead? Like (untested)

    var currentStatement = copyrightStatement.replace("(.*\\b)u'(\\d{4}\\b)".toRegex(), "\1\2")